src/test/scala/it/pagopa/interop/purposeprocess/RiskAnalysisValidationSpec.scala

```diff
@@ -12,103 +12,147 @@ import it.pagopa.interop.tenantmanagement.client.model.TenantKind
 import it.pagopa.interop.purposeprocess.api.impl.RiskAnalysisValidation
 import it.pagopa.interop.purposeprocess.api.impl.RiskAnalysisValidation.ValidationResult
 import it.pagopa.interop.purposeprocess.error._
-import it.pagopa.interop.purposeprocess.model._
 import org.scalatest.Assertion
 import org.scalatest.matchers.should.Matchers._
 import org.scalatest.wordspec.AnyWordSpecLike
+import it.pagopa.interop.purposeprocess.model.RiskAnalysisForm
 
 class RiskAnalysisValidationSpec extends AnyWordSpecLike {
   implicit val eqError: Eq[RiskAnalysisValidationError] = Eq.fromUniversalEquals
 
   "Risk Analysis Validation" should {
-    "succeed on correct form 1.0" in {
```

Comment:
⚠️ this was the test for the version 1.0 for PAs, it should not be migrated to 2.0.
The test for the PAs 2.0 should be a different test